### PR TITLE
Switch to `pnpm`

### PR DESCRIPTION
**Pull request by gossi**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public-hoist-pattern[]=*prettier*
+public-hoist-pattern[]=*eslint*
+public-hoist-pattern[]=!@typescript-eslint/*
+public-hoist-pattern[]=*ember-template-lint*
+public-hoist-pattern[]=*stylelint*
+strict-peer-dependencies=false
```
Comment (NullVoxPopuli) [inline .npmrc:6]: strict-peers needs to be true for all libraries, can you delete this file? thanks!!!!

File: package.json [hunk 2]
```diff
@@ -8,18 +8,16 @@
...
-    "test": "yarn workspaces run test"
+    "start:addon": "pnpm --filter ember-element-helper start",
+    "start:test": "pnpm --filter test-app start",
+    "lint": "pnpm --filter '*' lint",
+    "lint:fix": "pnpm --filter '*' lint:fix",
+    "test": "pnpm --filter '*' test"
   },
   "volta": {
     "node": "14.21.3",
-    "yarn": "1.22.19",
```
Comment (NullVoxPopuli) [inline package.json:22]: can you add pnpm here? :partying_face:

**Review by NullVoxPopuli — CHANGES_REQUESTED** [review]
this is _very_ close!!! nice work and thanks for working on this! left a couple comments

File: .npmrc (hunk 1 above)
Comment (gossi) [reply .npmrc]: done. Hoisting the lint packages is so far necessary for vscode plugins to properly work.

**Review by NullVoxPopuli — APPROVED** [review]
thank you!!

**Pull request merged**